src/pages/portfolio/Overview.tsx

```diff
@@ -73,7 +73,7 @@ export const Overview = () => {
         <AccountDetailsAndHistory />
       </DetachedSection>
 
-      {affiliatesEnabled && (
+      {affiliatesEnabled && dydxAddress && (
```

Comment:
[q] is this a bugfix? were people able to see the affiliates experience even if logged out

Reply:
yea this is a bug fix, it was behind the featureflag, so wasn't a production bug thankfully